src/components/IndiaStateMap/IndiaMapStats.js

```diff
@@ -1,47 +1,59 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import {motion} from 'framer-motion';
+import { motion } from 'framer-motion';
+import * as population from '../../data/Population.json';
 
-const IndiaMapStats = ({ stats, handleAttributeClick, activeAttribute}) => {
-  const {t} = useTranslation();
+const IndiaMapStats = ({ stats, handleAttributeClick, activeAttribute, mode, state }) => {
+  const { t } = useTranslation();
+
+  function calculateCases(cases) {
+    if (mode === 'normal')
+      return Number(cases);
+    let casesPerMillion = 0;
+    if (!(state === 'Sikkim' || state === 'Nagaland'))
+      casesPerMillion = Math.round((Number(cases) / population.default[state]) * 1000000);
+    return casesPerMillion + ' cases per million'
```

Comment:
Just return the `casesPerMillion` from here. Like this maybe

```suggestion
    return Number(casesPerMillion).toLocaleString('en')
```

This way we would not have to call `.toLocaleString('en')` in the jsx and it will look a bit cleaner.

Also, we can remove the `cases per million` text and then add this in the `<h6>` below. This will make sure we have 3 lines (in the boxes) on all devices and the box will not grow in size on bigger screens.

Reply:
Done